### ISSUE TITLE
QA - Cannot add alt text to Hero image

### ISSUE THREAD

**Issue by data-sync-user** [MozillaFoundation/foundation.mozilla.org#14295]
[https://docs.google.com/spreadsheets/d/1gE2WvOpEadRZAO51o2-OewZNgVynJv7RS_Rg3vKVpFY/edit?gid=259414541#gid=259414541](https://docs.google.com/spreadsheets/d/1gE2WvOpEadRZAO51o2-OewZNgVynJv7RS_Rg3vKVpFY/edit?gid=259414541#gid=259414541|smart-link) 

**Context**: Currently, the **Hero Section > Hero Image** block does **not allow editors to add an image alt description**. This limits accessibility and prevents proper semantic tagging of images, which is especially important for screen readers and SEO.

**Page this impacts**: General Page using the Hero Section > Hero Image block

### **How to replicate:**
1. Edit any page with a **Hero Section** in Wagtail
1. Add or modify a **Hero Image**
1. Observe that there is **no field** to enter an **alt description** for the image
1. Attempting to save the image without alt text is allowed, but **alt cannot be added manually**

### **Request:**
Update the **Hero Image block** to include an **optional “Alt text” field**, allowing editors to provide descriptive alt content for accessibility and semantic structure.

!image-20250721-164158.png|width=254,height=234,alt="image-20250721-164158.png"!

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-2984)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/90551">image-20250721-164158.png</a>

**Comment by data-sync-user** [comment]
➤ Kevin Duong commented:

Hi Rob DiVincenzo - This relates to https://mozilla-hub.atlassian.net/browse/TP1-2990 ( https://mozilla-hub.atlassian.net/browse/TP1-2990|smart-link ). I’m not seeing an option to add alt text to a Hero image? See example page here https://www.mozillafoundation.org/cms/pages/31559/edit/ ( https://www.mozillafoundation.org/cms/pages/31559/edit/ )

**Comment by data-sync-user** [comment]
➤ Adalberto Cima commented:

Hi team
Some updates on this task that include a change in the field type for hero_image, this will allow us to use the images in different contexts and be able to add the description using alt text.

  * PR for this task here ( https://github.com/MozillaFoundation/foundation.mozilla.org/pull/14451 )
  * Test page can be also found here:
https://foundation-s-tp1-2984-c-ju6m6u.herokuapp.com/cms/pages/448/edit/ ( https://foundation-s-tp1-2984-c-ju6m6u.herokuapp.com/cms/pages/448/edit/ )
https://foundation-s-tp1-2984-c-ju6m6u.herokuapp.com/en/inspire_test/ ( https://foundation-s-tp1-2984-c-ju6m6u.herokuapp.com/en/inspire_test/|smart-link ) 
Login: admin
Password: xpn*rBF)W)WF^9Xr

Kevin Duong Genevieve Baldassano 

Thanks

**Comment by data-sync-user** [comment]
➤ Adalberto Cima commented:

Hi team closing this, since the merge/deploy are completed.

ccSimon Acosta Torres
